app/assets/stylesheets/govuk-component/_button.scss

```diff
@@ -0,0 +1,41 @@
+.pub-c-button {
```

Comment:
Do we need to document what `pub-c` means? As it stands this is different to the other static components.